app/javascript/styles.scss

```diff
@@ -0,0 +1,5 @@
+@import "~bulma/bulma";
+
+$fa-font-path: "@fortawesome/fontawesome-free/webfonts";
+@import "@fortawesome/fontawesome-free/scss/fontawesome";
```

Comment:
I dropped bulma & fontawesome from the pack, can you load this from via [CDN](https://cdnjs.com/libraries/bulma-tooltip)?